requirements.txt

```diff
@@ -1,11 +1,34 @@
 -i https://pypi.org/simple
-click==8.1.7; python_version >= '3.7'
+altgraph==0.17.4
+astroid==3.0.3 ; python_full_version >= '3.8.0'
+autopep8==2.0.4
+build==1.0.3
+coverage==7.4.1
+dill==0.3.8 ; python_version < '3.11'
+exceptiongroup==1.2.0 ; python_version < '3.11'
+iniconfig==2.0.0 ; python_version >= '3.7'
+isort==5.13.2 ; python_full_version >= '3.8.0'
+mccabe==0.7.0 ; python_version >= '3.6'
+packaging==23.2 ; python_version >= '3.7'
+platformdirs==4.2.0 ; python_version >= '3.8'
+pluggy==1.4.0 ; python_version >= '3.8'
+pycodestyle==2.11.1 ; python_version >= '3.8'
+pyinstaller==6.4.0
+pyinstaller-hooks-contrib==2024.1 ; python_version >= '3.7'
+pylint==3.0.3
+pyproject-hooks==1.0.0 ; python_version >= '3.7'
+pytest==8.0.0
+pytest-cov==4.1.0
+setuptools==69.0.2
+tomli==2.0.1 ; python_version < '3.11'
+tomlkit==0.12.3 ; python_version >= '3.7'
+typing-extensions==4.9.0 ; python_version >= '3.8'
+click==8.1.7 ; python_version >= '3.7'
 colorama==0.4.6
-markdown-it-py==3.0.0; python_version >= '3.8'
-mdurl==0.1.2; python_version >= '3.7'
-pygments==2.17.2; python_version >= '3.7'
+markdown-it-py==3.0.0 ; python_version >= '3.8'
+mdurl==0.1.2 ; python_version >= '3.7'
+pygments==2.17.2 ; python_version >= '3.7'
 rich==13.7.0
 shellingham==1.5.4
-tabulate==0.9.0; python_version >= '3.7'
-typer[all]==0.9.0; python_version >= '3.6'
-typing-extensions==4.9.0; python_version >= '3.8'
+tabulate==0.9.0
+typer[all]==0.9.0
```

Comment:
Something does not look right here.

Could you just run: `make requirements` and push it up. Should fix this.